includes/class-wsuwp-people-directory-page-template.php

```diff
@@ -446,23 +449,40 @@ public function save_post( $post_id ) {
 			}
 		}
 
-		// Set a flag to flush rewrite rules.
-		if ( false === apply_filters( 'wsuwp_people_default_rewrite_slug', false ) ) {
-			set_transient( 'wsuwp_people_directory_flush_rewrites', true );
-		}
-
 		// Update associated people data.
 		if ( ! isset( $_POST['_wsu_people_directory_profile_ids'] ) ) {
 			return;
 		}
 
 		// Stop here if the order of people hasn't changed.
-		//if ( $previous_ids === $_POST['_wsu_people_directory_profile_ids'] ) {
-		//	return;
-		//}
+		if ( $previous_ids === $_POST['_wsu_people_directory_profile_ids'] ) {
+			return;
+		}
 
-		// Save order data of the associated people.
-		$ids = explode( ' ', $_POST['_wsu_people_directory_profile_ids'] );
+		// Set a flag to flush rewrite rules.
+		if ( false === apply_filters( 'wsuwp_people_default_rewrite_slug', false ) ) {
+			set_transient( 'wsuwp_people_directory_flush_rewrites', true );
+		}
+
+		$scheduled_event_args = array(
+			$_POST['_wsu_people_directory_profile_ids'],
```

Comment:
I'm a little worried about this not being sanitized/verified before being passed to the callback. It looks like we weren't doing it before, but it might be good to parse these as `int` first.

Reply:
Good call! I think sanitizing it here makes the most sense.